openpype/hosts/max/startup/startup.ms

```diff
@@ -7,6 +7,9 @@
 
     local pythonpath = systemTools.getEnvVariable "MAX_PYTHONPATH"
     systemTools.setEnvVariable "PYTHONPATH" pythonpath
+    
+    # opens the create menu on startup to ensure users are presented with a useful default view.
+    max create mode
```

Comment:
Don't know much about 3dsmax - but I have a feeling this might be relatively user-preference or studio-specific? Why wouldn't someone want max to start up with their own "view" or "layout" - assuming it's referring to the layout that Max starts in?

Reply:
You may find the changelog description useful on this. This command is to make create mode active and it is used in this particular maxscript to make sure the panel on the right-hand side of 3dsmax aligns with manually launching 3dsmax when you launch 3dsmax with AYON plugins. In the develop branch, it launches with the panel where locates the utility part.